packages/editor/.env.local

```diff
@@ -1,4 +1,4 @@
 ORIGIN=localhost:4000
-REACT_APP_INFURA_API_KEY=58b57895912346bf8aefba3cab0db459
-REACT_APP_AMPLITUDE_API_KEY=5824e3f0eac5febce797022a39309f01
-REACT_APP_API_BASE_URL=https://studio-api-staging.ethereum.org
+INFURA_API_KEY=58b57895912346bf8aefba3cab0db459
+AMPLITUDE_API_KEY=5824e3f0eac5febce797022a39309f01
+API_BASE_URL=https://studio-api-staging.ethereum.org
```

Comment:
Can't we remove the sensite api keys from here? Are they neede for local dev? Also is INFURA_API_KEY still required for anything?

Reply:
Also I would rename this into `.env` instead of `.env.local` I think

Reply:
Infura is not needed for anything right now, so I've removed it. But in case we add the feature for other networks, this change will need to be reverted.